.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: CMake
+
+on:
+  push:
+    branches:
+      - live-debug*
+      - ci_test
```

Comment:
```suggestion
      - master
```

Reply:
I think it is a good idea to leave master out, since push to master is only allowed with PRs, which are already tested.
having a special `ci_test` is debatable, the idea is to have a special branch so that we run CI every time we push, which is useful mostly to develop the CI itself. Do you suggest a better way?

Reply:
I'd say it's standard practice to run CI against `master`. It's true it would virtually be redundant wrt PRs, regardless of merge strategy. Additional steps can be done if `isMaster()` like storing artifacts, publishing documentation, displaying CI badges and so on. I guess we can look at it also as a sanity check. 

> having a special `ci_test` is debatable, the idea is to have a special branch so that we run CI every time we push, which is useful mostly to develop the CI itself. Do you suggest another way? @alexsavulescu

The CI will run for each PR every time a new commit is pushed even when working on the CI, so there is no need for a special branch. To work on the CI itself, the most straightforward (and isolated way) is to use a fork, where you can modify `master` directly or even use PRs (if you don't want to add `master` branch on push).